doc/classes/ProjectSettings.xml

```diff
@@ -2686,11 +2686,9 @@
 		</member>
 		<member name="rendering/shading/overrides/force_vertex_shading" type="bool" setter="" getter="" default="false">
 			If [code]true[/code], forces vertex shading for all rendering. This can increase performance a lot, but also reduces quality immensely. Can be used to optimize performance on low-end mobile devices.
-			[b]Note:[/b] This setting currently has no effect, as vertex shading is not implemented yet.
 		</member>
 		<member name="rendering/shading/overrides/force_vertex_shading.mobile" type="bool" setter="" getter="" default="true">
```

Comment:
PS: I think we should default this to `false` (that is, removing the `mobile` override entirely) because this will cause a behavior change between 4.2 and this PR.

While forcing vertex shading improves performance significantly, it also breaks visuals on most 3D projects in an unexpected manner. Forcing vertex shading also doesn't address some of the bottlenecks like fillrate, while [3D resolution scaling](https://github.com/godotengine/godot/pull/73997) addresses this. In other engines, defaulting to a lower 3D resolution scale is a more popular approach to scaling a project to mobile platforms compared to forcing a setting that changes materials' appearance.

Reply:
defaulting to false is fine, but I guess we should still have `mobile` override, because some project may need this only for mobile platforms.

Reply:
The `mobile` override can be added manually by users. We don't need to keep it if its default value is identical to the non-overridden value :slightly_smiling_face:

Reply:
Oh, Right totally forgot about it in the project settings.